### DISCUSSION TITLE
What do you want to see improved the most?

### DISCUSSION THREAD

**Discussion by Alek99** [reflex-dev/reflex#3903] [Polls]
Hi everyone doing a quick poll to see what people want improved the most.

**Comment by bertbarabas** [comment]
1. Data Editor - Complete wrapping all properties.  It's an awesome component but frustrating to not have all it's abilities available.
2. File upload performance (it's very slow and consumes a lot of CPU)

**Reply by Alek99** [reply]
That's for the feedback that's super helpful to know.

**Reply by noirsender** [reply]
1. general improvement for wrapping react components: maybe some examples how to wrap complex libs like swapy

2. drag and drop combined with docking manager 

3. full fledged editor component with all the bells and whistles

4) learning documentation

5) typing in the helper llm is very slow and the llm itself very limited: maybe some better llm models + better rag for all reflex material and repos

6) webassembly easy integration

**Reply by Alek99** [reply]
Thanks for the feedback!

**Reply by h4gen** [reply]
I would second the DataEditor. It is such an awesome tool and we barely scratch the surface rn. I find it especially frustrating that not all cell kinds are available b/c `kind` is no param but the cell is configured via mapping from the dtype.

**Comment by itsmeadarsh2008** [comment]
Regarding the survey,
these are the issues which I have entangled the most.
#3616 (for seamless deployment)
#3759
#3765 (maybe unnecessary but can reduce manual work for editing tailwind config)
#3904

**Comment by Chatterc** [comment]
I would like a UI/UX designer tool that allows me to drag and drop reflex components to speed my front end development. Tool should like to my project and commit new .py files as I add new components/state/pages to the project.

**Reply by Alek99** [reply]
Currently a drag and drop editor is not on the roadmap but we are working on something that should help a lot with the frontend development side of things. Join discord and see the channel flexgen for context

**Comment by jq6l43d1** [comment]
I would like the reverse compiler to work https://flexgen.reflex.run/reverse-compiler/ I'd also like it to be open source so that I can run it myself.

**Comment by PasqualePuzio** [comment]
Drag and Drop would be great and useful for so many use cases

**Reply by dentroai** [reply]
drag and drop what? You can already do that with the file upload component

**Comment by dentroai** [comment]
Easier / automatic way to wrap react packages or more guides on that.

**Comment by s4n-cz** [comment]
Upgrade to Pydantic 2.0 #1539
